windows/vbs_enable_disable/vbs_enable_disable.yml

```diff
@@ -29,6 +29,24 @@
             skip_reason: "Not Applicable"
           when: vm_firmware | lower != 'efi'
 
+        - name: "Set ESXi CPU model"
+          ansible.builtin.set_fact:
+            esxi_cpu_model: |-
```

Comment:
It looks like `vendor` or `manufacturer` is more proper than `model` at here.

Reply:
I thinks "model" is more general. Here the focus is on CPU architecture not vendor.